e2e/cypress/integration/team_settings/join_closed_team_with_not_allowed_email_spec.js

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @team_settings
+
+import {getRandomId} from '../../utils';
+import * as TIMEOUTS from '../../fixtures/timeouts';
+
+describe('Team Settings', () => {
+    const randomId = getRandomId();
+    let testTeam;
+
+    before(() => {
```

Comment:
Also, could you add update to config and disable guest account and LDAP login? I saw this failing when running on licensed version.
```javascript
        cy.apiUpdateConfig({
            GuestAccountsSettings: {
                Enable: false,
            },
            LdapSettings: {
                Enable: false,
            },
        });
```

Reply:
I added this after `cy.apiInitSetup` . Is this okay or should I move it ?

Reply:
I think it's fine but would request to move before `cy.apiInitSetup` for consistency.  Thanks for updating.